products/xcp-ng.md

```diff
@@ -0,0 +1,63 @@
+---
+title: XCP-ng
+category: os
+iconSlug: xcp-ng
+permalink: /xcp-ng
+alternate_urls:
+-   /xcpng
+versionCommand: software-version
+releasePolicyLink: https://xcp-ng.org/docs/releases.html#all-releases
+
```

Comment:
Do you know if there is a page listing versions with their release date ? This would make it possible to automatically update `latest` releases.

Reply:
This should work : https://github.com/xcp-ng/xcp-ng-release/tags.